tests/test_service.py

```diff
@@ -0,0 +1,127 @@
+
+import pytest
+import logging
+
+from plumbum import ProcessExecutionError, local
+from plumbum.cmd import docker
+from plumbum.machines.local import LocalCommand
+
+logger = logging.getLogger()
+
+CONTAINER_NAME = "docksockprox_test"
+SOCKET_PROXY = "127.0.0.1:2375"
```

Comment:
You cannot have 2 containers with the same name, or reading the same port. We want to be able to run tests on parallel, so please think if this is going to produce collisions, and if so, use dynamic names and ports.